pom.xml

```diff
@@ -61,15 +61,82 @@
             <artifactId>liquibase-core</artifactId>
             <version>${liquibase.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.opensearch.client</groupId>
+            <artifactId>opensearch-java</artifactId>
+            <version>2.10.3</version>
```

Comment:
@rursprung I will try to cut the new client release today (https://github.com/opensearch-project/opensearch-java/issues/1011)

Reply:
https://github.com/opensearch-project/opensearch-java/releases/tag/v2.10.4

Reply:
thanks! i've now updated to that and added the exclusion again. testing works locally (with the snapshot of `liquibase-core`)